README.md

```diff
@@ -6,35 +6,81 @@
 Pecas is a time tracking leaderboard for
 [https://nokotime.com](https://nokotime.com).
 
-## Setup
+## Getting started
 
-To install Pecas in a development environment, you can follow the next steps:
+To get started with the app, clone the repo and then install the needed gems running the setup script:
 
-### Requirements
+```bash
+git clone git@github.com:fastruby/pecas.git
+cd pecas
+./bin/setup
+```
 
-- Docker
-- Docker-Compose
-- Git
+## Environment Variables
 
-### First-time only
+The command `./bin/setup` between other things creates the `.env` file which contains the following env variables:
 
-    git clone git@github.com:fastruby/pecas.git
-    cd path/to/pecas
-    docker-compose build
-    docker-compose run web /bin/bash
-    ./bin/setup
+```yml
+# .env.sample
+NOKO_ACCOUNT_HOST=ombulabs
+NOKO_TOKEN=foobar
+SMTP_SERVER=smtp.sendgrid.net
+SMTP_PORT=587
+SMTP_DOMAIN=ombushop.com
+SMTP_USER_NAME=ernesto@ombushop.com
+SMTP_USER_PASSWORD=secret
+COUNTRY_CODE="ar"
+DATABASE_NAME="pecas"
+BASIC_AUTH_NAME="user"
+BASIC_AUTH_PASSWORD="secret"
+```
 
-You must setup your `NOKO_TOKEN` in the `.env` file. You can fetch the token value from the Noko app. After logging in, you can find the API token or create a new API token under the `Connected Apps` section. You can setup your
-`COUNTRY_CODE` environment variable with an ISO 3166 country code. Otherwise
-the emails will be sent on holidays.
+Now, you just need to update the `NOKO_TOKEN` env with the correct value.
+if the `.env` file already exist, then you need to manually add these variables to the file with correct value.
```

Comment:
@JuanVqz I think we can remove this sentence. Sounds a bit redundant to me

Reply:
The paragraph should probably look like this:
```
Now, you just need to update the `NOKO_TOKEN` env with the correct value. You can fetch the token value from the Noko app, after logging in, you can find the API token or create a new one under the `Connected Apps > Noko API > Personal Access Tokens section`.
```

Reply:
yep, I thought the same. done.